CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+cmake_minimum_required(VERSION 3.6)
+project(Gromit_MPX)
+
+set(target_name gromit-mpx)
+set(version 1.2)
+
+configure_file(build-config.h_cmake_in build-config.h)
+
+find_package(PkgConfig)
+
+pkg_check_modules(gtk3 "gtk+-3.0 >= 2.99.3")
+pkg_check_modules(appindicator3 "appindicator3-0.1 >= 0.4.92")
+pkg_check_modules(xinput "xi >= 1.3")
+pkg_check_modules(x11 x11)
+
+include_directories(
+    ${CMAKE_CURRENT_BINARY_DIR}
+    ${gtk3_INCLUDE_DIRS}
+    ${appindicator3_INCLUDE_DIRS}
+    ${ximput_INCLUDE_DIRS}
+    ${x11_INCLUDE_DIRS}
+)
+
+link_directories(
+    ${gtk3_LIBRARY_DIRS}
+    ${appindicator3_LIBRARY_DIRS}
+    ${ximput_LIBRARY_DIRS}
+    ${x11_LIBRARY_DIRS}
+)
+
+set(sources
```

Comment:
-g is automatically added to Debug builds. -O3 -DNDEBUG is automatically added to Release builds.